src/WebformCivicrmPostProcess.php

```diff
@@ -2406,6 +2406,10 @@ private function fillDataFromSubmission() {
           if (!empty($this->data[$ent][$c][$table][$n][$name]) && is_array($this->data[$ent][$c][$table][$n][$name])) {
             $val = array_unique(array_merge($val, $this->data[$ent][$c][$table][$n][$name]));
           }
+          if (substr($name, 0, 6) === 'custom' || ($table == 'other' && in_array($name, ['group', 'tag']))) {
+            $val = array_filter($val);
+          }
+
```

Comment:
Note to myself to see how this works with tagsets -> field keys are like `civicrm_1_contact_1_other_tag_14`

Reply:
@maynardsmith - this 👆 is where the checkboxes get filtered. See if you work tagsets into this. Then we can remove the temporary ad-hoc override -> https://github.com/colemanw/webform_civicrm/blob/8.x-5.x/src/WebformCivicrmPostProcess.php#L932-L937